<launch>
    <node pkg="zeabus_hydrophone" type="hydrophone_ros_node.py" name="hydrophone" respawn="true">
        <param name="PortName" value="/dev/ttyACM0"/>
        <param name="PingerFrequency" value="37500"/>        
        <param name="LowestFrequency" value="20000"/>        
        <param name="HighestFrequency" value="45000"/>        
        <param name="FrontThreshold" value="0.3"/>
        <param name="PowerThreshold" value="0.02"/>
        <param name="DelayObserve" value="2000"/>
        <param name="AmplifierNormalizedGain" value="0.2"/>
    </node>
</launch>
